src/services/room-service.js

```diff
@@ -77,4 +77,9 @@ export default class RoomService {
   getRoomById(roomId) {
     return this.roomStore.findOne({ _id: roomId });
   }
+
+  async isRoomMemberOrOwner(roomId, userId) {
+    const room = await this.roomStore.findOne({ $or: [{ '_id': roomId, 'members.userId': userId }, { _id: roomId, owner: userId }] });
```

Comment:
To me it would be more logical to do 
```
{ $and: [{ _id: roomId }, { $or: [{ 'members.userId': userId }, { owner: userId }] }] }
``` 
and would also remove the duplication.

I would not insist though, but if you keep the duplication then I'd ask to please make it consistent and go with either `'_id': roomId` or `_id: roomId`

Reply:
Changed